exporters/otlp/otlpmetric/otlpmetrichttp/internal/oconf/options.go

```diff
@@ -287,9 +287,7 @@ func WithEndpointURL(v string) GenericOption {
 
 		cfg.Metrics.Endpoint = u.Host
 		cfg.Metrics.URLPath = u.Path
-		if u.Scheme != "https" {
-			cfg.Metrics.Insecure = true
-		}
+		cfg.Metrics.Insecure = u.Scheme != "https"
```

Comment:
See the header of the file

```
// Code created by gotmpl. DO NOT MODIFY.
// source: internal/shared/otlp/otlpmetric/oconf/options.go.tmpl
```

Reply:
I also realised there is the same issue for grpc exporter. Yes, I can do all fixes. Sorry, haven't checked the header :)

Reply:
No problem. Thanks for the PR 👍

Reply:
looks like otlplog doesn't have same issue https://github.com/open-telemetry/opentelemetry-go/blob/506a9baf5e00147118caea839b529e0f6539e93d/exporters/otlp/otlplog/otlploghttp/config.go#L186